packages/calling/src/common/Utils.ts

```
@@ -322,6 +338,30 @@ export async function handleRegistrationErrors(
+      emitterCb(lineError, finalError);
+      break;
+    }
+
+    case ERROR_CODE.TOO_MANY_REQUESTS: {
+      const caller = loggerContext.method || 'handleErrors';
+
+      if (err.headers) {
+        const retryAfter = Number(err.headers['retry-after']);
+        retry429CB(retryAfter, caller);
+      }
+    }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)